libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteLine.kt

```diff
@@ -1392,9 +1457,21 @@ internal class MapRouteLine(
                 )
 
                 val routeFeature = when (identifier) {
-                    null -> Feature.fromGeometry(routeGeometry)
-                    else -> Feature.fromGeometry(routeGeometry).also {
-                        it.addBooleanProperty(identifier, true)
+                    null -> {
+                        Feature.fromGeometry(
+                            routeGeometry,
+                            null,
+                            Random.nextInt().toString()
```

Comment:
I know that probability is low, but I'd rather control the values than trust the generator to not output duplicates. Could you introduce a set of IDs base on the number of available routes and safely assign them?